dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Messages.properties

```diff
@@ -48,7 +48,14 @@ WIZARD_PAGE_SELECT_GENERATOR_TYPE=Select a generator type.
 BROWSE_DIALOG_MESSAGE_SELECT_DEFINITION=Select an OpenAPI definition from the project.
 BROWSE_DIALOG_TITLE_SELECT_DEFINITION=Select an OpenAPI definition file.
 
-BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder for the generated files.
-BROWSE_DIALOG_TITLE_SELECT_FOLDER=Select a folder.
+BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder under the current project
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID=Output Folder Invalid
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID_MESSAGE=The selected folder must be the project root folder or one of project's child folders.
 
-BROWSE_DIALOG_NO_CHILD_FOLDERS=This project does not contain child folders. The stub will be generated into the project root.
+MAVEN_PROJECT_DETECTED_TITLE=Overwrite Existing Files
+MAVEN_PROJECT_DETECTED_DESCRIPTION=Warning: Existing files might be overwritten and the existing pom.xml will be merged with the generated pom.xml from the code generator. The original pom.xml will be backed up. Do you want to continue?
```

Comment:
Suggested rewrite:
Warning: Existing files might be overwritten when the existing pom.xml file is merged with the generated pom.xml file from the code generator. The original pom.xml file will be backed up. Do you want to continue?

Reply:
I want to convey that files other than the pom.xml will be affected.  Those other files might be overwritten.  The pom.xml file is different in that it will be merged with the one that is generated.

Reply:
How about...
Existing files might be overwritten when the existing pom.xml file is merged with the generated pom.xml file from the code generator. The original pom.xml file will be backed up, but other files will not be. Do you want to continue?

Reply:
They are independent situations, Existing files might be overwritten AND the existing pom.xml file is merged.

Reply:
How about...
The existing pom.xml file will be backed up when it is is merged with the generated pom.xml file from the code generator. However, other existing files might be overwritten. Do you want to continue?

Reply:
Acceptable.  Will make this change.